README.md

```diff
@@ -263,16 +258,14 @@ Place your ias api key and your SPID in the ``ias`` folder as follows:
     echo 'YOUR_API_KEY' > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/config/ias/api_key.txt
     echo 'YOUR_SPID' > ${GOPATH}/src/github.com/hyperledger-labs/fabric-private-chaincode/config/ias/spid.txt
 
-
-### Fabric Private Chaincode
+#### Fabric Private Chaincode
 
 Clone the code and make sure it is on your `$GOPATH`. (Important: we assume in this documentation and default
```

Comment:
I think here we could add a hint that the docker container already has the code and this step is not needed in tthat case